### PR TITLE
fix error with filters

### PR DESCRIPTION
**Pull request by almas50**
module broke due to line domain="[]"

### CODE REVIEW

File: project_task_subtask/views/project_task_subtask.xml [hunk 1]
```diff
@@ -102,7 +102,7 @@
         </field>
     </record>
     <record id="project_task_subtask_view_search" model="ir.ui.view">
-        <field name="name">project.task.subtask.filter</field>
+       <field name="name">project.task.subtask.filter</field>
```
Comment (Enigma228322) [inline project_task_subtask/views/project_task_subtask.xml]: @almas50

**Comment by Enigma228322** [comment]
@yelizariev @trojikman I checked it. It seems ok.

**Comment by trojikman** [comment]
> @yelizariev @trojikman I checked it. It seems ok.

Ok, I'll check it soon

File: project_task_subtask/views/project_task_subtask.xml [hunk 2]
```diff
@@ -114,24 +114,11 @@
...
-                        string="User"
-                        domain="[]"
-                        context="{'group_by':'user_id'}"
-                    />
-                    <filter string="State" domain="[]" context="{'group_by':'state'}" />
-                </group>
+                 <group expand="0" string="Group By">
+                     <filter string="User" name="user" context="{'group_by':'user_id'}"/>
+                     <filter string="Project" name="project" context="{'group_by':'project_id'}"/>
+                     <filter string="Task" name="task" context="{'group_by':'task_id'}"/>
```
Comment (yelizariev) [inline project_task_subtask/views/project_task_subtask.xml]: Keep the same order, please: Project, Task, User

**Review by yelizariev — COMMENTED** [review]
You need to add your copyright line at project_task_subtask/views/project_task_subtask.xml at least.
More information is here: https://gitlab.com/itpp/handbook/-/blob/master/technical-docs/copyrights.md

**Review by yelizariev — APPROVED** [review]

**Review by Enigma228322 — APPROVED** [review]

**Review by trojikman — APPROVED** [review]

**Pull request closed without merging**